subsys/bluetooth/host/iso.c

```diff
@@ -1048,14 +1072,24 @@ static struct net_buf *hci_le_set_cig_params(const struct bt_iso_cig *cig,
 	return rsp;
 }
 
+static struct bt_iso_cig *get_cig(const struct bt_iso_chan *iso_chan)
+{
+	if (iso_chan->iso == NULL) {
+		return NULL;
+	}
+
+	return &cigs[iso_chan->iso->iso.cig_id];
```

Comment:
Should there be a array out-of-bounds check too?

Reply:
Arguably yes. I'll add an assert since this should not happen unless something else have gone wrong.